.github/workflows/code-engine-preview.yml

```diff
@@ -116,6 +119,17 @@ jobs:
             IBMID_CLIENT_SECRET=${{ secrets.ibmid_client_secret }}
             MONGODB_URI=${{ secrets.mongodb_uri }}
 
+      - name: Build and push to IBM registry for saiba
+        if: github.repository == 'Qiskit/saiba'
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: ${{ env.DOCKER_IMAGE_TAG }}
+          cache-from: type=registry,ref=${{ env.DOCKER_IMAGE_TAG }}
+          cache-to: type=inline
+          build-args: |
+            PUBLIC_API_BASE=inputs.public_api_base
```

Comment:
Probably we don't need this if not that taking into account the `environment` the url will be set in `nuxt.config` [configuration file](https://github.com/Qiskit/saiba/blob/main/nuxt.config.ts#L40)?

Reply:
the default in `nuxt.config` is `http://localhost:8055`? are you suggesting we change this default?

Reply:
I think it should be `NUXT_PUBLIC_API_BASE`.

Reply:
i set it to `NUXT_PUBLIC_API_BASE` in the dockerfile: https://github.com/Qiskit/saiba/blob/9e1c8514cb56b4fd20af454ba23f0e5c52838779/Dockerfile#L22
but yeah, i should probably keep it consistent throughout

Reply:
Ooh, got it! Your are right.
No problem from my side to keep it as it is.

Reply:
took approach suggested by @Tansito so no longer need this PR